packages/react/src/components/FluidDatePickerInput/FluidDatePickerInput.tsx

```diff
@@ -11,11 +11,11 @@ import { FormContext } from '../FluidForm/FormContext';
 
 const FluidDatePickerInput = React.forwardRef(function FluidDatePickerInput(
   { ...other },
-  ref
+  ref: React.Ref<HTMLDivElement>
 ) {
   return (
     <FormContext.Provider value={{ isFluid: true }}>
-      <DatePickerInput ref={ref} {...other} />
+      <DatePickerInput id={''} labelText={undefined} ref={ref} {...other} />
```

Comment:
Could the `id` and `labelText` be removed here so users can still configure them?

Reply:
id and labelText are required props in DatepickerInput so need to pass something. What can we do other than this?

Reply:
@riddhybansal If they're required in `DatePickerInput`, they should be required here in `FluidDatePickerInput` too and passed through `...other`.

If typescript isn't automatically inferring that relationship then I think we need to explicitly specify that `FluidDatePickerInput` inherits `DatePickerInputProps`.